test/common/index.js

```diff
@@ -812,7 +812,49 @@ function restoreWritable(name) {
   delete process[name].writeTimes;
 }
 
+function onResolvedOrRejected(promise, callback) {
```

Comment:
One hackish way around that is to set the name property on the function after it is defined. E.g. 

```js
function onResolvedOrRejected(promise, callback) { /* ... */ }
onResolvedOrRejected.name = 'finally';
```

But I definitely do not see that as being critical.